src/core/variable.jl

```diff
@@ -127,6 +128,68 @@ function variable_reactive_line_flow(pm::GenericPowerModel; bounded = true)
     return q
 end
 
+############## DC Lines ############################################
+function variable_line_flow_dc(pm::GenericPowerModel; kwargs...)
+    variable_active_line_flow_dc(pm; kwargs...)
+    variable_reactive_line_flow_dc(pm; kwargs...)

```

Comment:
Following the naming convention would be `variable_active_dc_line_flow(sol, pm)`